plugins/woocommerce/tests/e2e-pw/tests/merchant/order-refund.spec.js

```diff
@@ -76,9 +84,18 @@ test.describe.serial( 'WooCommerce Orders > Refund an order', () => {
 			'9.99'
 		);
 		await expect( page.locator( '#refund_amount' ) ).toHaveValue( '9.99' );
-		await expect( page.locator( '.do-manual-refund' ) ).toContainText(
-			`Refund ${ currencySymbol }9.99 manually`
-		);
+
+		if (LANGUAGE === 'ar_AR') {
+			await expect( page.locator( '.do-manual-refund' ) ).toContainText(
+				'مبلغ مُستردّ $9.99 يدويًا'
+			);
+		} else {
+			await expect( page.locator( '.do-manual-refund' ) ).toContainText(
+				`${ getTranslationFor('Refund') } ${ currencySymbol }9.99 ${
+					getTranslationFor('manually')
+				}`
+			);
+		}
```

Comment:
That's correct, due to the formatting in the RTL languages, the formatting is different so this one had to be handled slightly differently. 

However, on reflection, I don't like the if-else here and am going to pull the translation out to be something more readable like:

await expect( page.locator( '.do-manual-refund' ) ).toContainText(
				getTranslationFor('Refund 9.99 manually')
			);

This will remove the need for the if-else

Reply:
I've just spotted there is already a discussion to remove the if-else here, I added another suggestion in another code related to this, please feel free to decline it if this approach looks better!